### ISSUE TITLE
Can you clarify what devices are compatible?

### ISSUE THREAD

**Issue by nheeren** [vfilimonov/co2meter#18]
I just found this fantastic project. Great work. I would love to use it with my Raspberry Pi at home. 

However, from the README I am unclear as to what meters are compatible. Is this repository tailored to devices from the brand _CO2meter.com_ or something?

Maybe you can loose a word or two in the README. Thanks!

**Comment by vfilimonov** [comment]
Hello @nheeren , thanks for your interest in the project.

It is not an official library for any of the brands. I've coded the library for my particular meter ([link to russian webstore](https://shop.dadget.ru/catalog/house/detektor_uglekislogo_gaza/)). The protocol looked to be rather standard, so I hacked around the code I've found in the internet (see links in readme). And it turned out to be working for some other brands as well. But since I do not own any of other CO2 meters, I could not comment whether it will or will not work there.
